cms/src/routes/images.js

```diff
@@ -18,6 +23,26 @@ conn.once('open', function() {
   console.log('GridFSBucket is ready for uploads');
 });
 
+const s3 = new aws.S3({
```

Comment:
Can we move the S3 config to a module of its own in the /services directory? We can import it here for use.